test/functional/cmdLineTests/cmdLineTest_J9tests/j9tests_Java8.xml

```diff
@@ -550,7 +550,8 @@
   <command>$EXE$ -Xdump:none -Xmx7g -Djava.lang.stringBufferAndBuilder.growAggressively -cp $Q$$JARPATH$$Q$ TestStringBufferAndBuilderGrowth</command>
   <output regex="no" type="success">StringBuffer capacity=2147483647 StringBuilder capacity=2147483647</output>
   <output regex="no" type="success">Option too large</output>
- </test>
+  <output regex="no" type="success">Not enough resource to run test</output>
+</test>
```

Comment:
Wasn't the indentation of this correct before?

Reply:
Fixed.